drush_config_import/drush_config_import.php

```diff
@@ -5,5 +5,5 @@
 echo "Import of configuration complete.\n";
 //Clear all cache
 echo "Clearing cache.\n";
```

Comment:
Should likely update the log accordingly too. It's now will "Rebuild the cache" rather then "Clear all cache".